src/views/Leads/Table/index.js

```diff
@@ -266,7 +331,7 @@ export default class Table extends React.Component {
                             } = emptyObject,
                             leadPatchRequest,
                         },
-                    } = this.props;
+                    } = props;
```

Comment:
defined leadPatchRequest outside of this scope and add them to dependency individually